_posts/2018-02-16-Java-Time2.md

```diff
@@ -0,0 +1,1564 @@
+
+{% highlight java %}
+public class NetworkDomainHandler {
+    ...
+    public List<Domain> fetchAllDomainsForEnterprise(Enterprise enterprise) {
+        DomainsFetcher fetcher = enterprise.getDomains(); // 11
+        List<Domain> list = new ArrayList<>();
+
+        try {
+            if (fetcher.count() > 0) { // 12
+                list = fetcher.get(); // 13
+            }
+        } catch (RestException e) {
+            e.printStackTrace();
+        }
+
+        return list;
+    }
+    
+    public void deleteDomain(Domain domain) { // 14
+        if (domain == null) {
+            System.out.println("Domain does not exist");
+        } else {
+            System.out.println("Domain " + domain.getName() + " will be deleted");
+            try {
+                domain.delete(); // 15
+            } catch (RestException e) {
+                e.printStackTrace();
+            }
+        }
+    }
+{% endhighlight %}
+
+In Line 11 of the `fetchAllDomainsForEnterprise` method, we gather all Domain children for a given Enterprise into a Fetcher.  Then in Lines 12 and 13, we check whether there are any domains and when any are found, they are added to a Java list.
+
+Line 14 presents the `deleteDomain` method which performs the actual deletion of a Domain object from VSD.  If the Domain object is not initialized, then the method simply prints a message.  Otherwise, at Line 15, the Domain will be deleted.
+
+# Removing a Domain Template
+The `NetworkDomainTemplateHandler` class can fetch all sibling Domain Template objects that belong to the same Enterprise, and can delete a specific Domain Template instance.  Here are the relevant code snippets from that class:
+
+{% highlight java %}
+public class NetworkDomainTemplateHandler {
+    ...
+    public List<DomainTemplate> fetchAllTemplatesForEnterprise(Enterprise enterprise) {
+        DomainTemplatesFetcher fetcher = enterprise.getDomainTemplates(); // 9
+        List<DomainTemplate> list = new ArrayList<>();
+
+        try {
+            if (fetcher.count() > 0) { // 10
+                list = fetcher.get(); // 11
+            }
+        } catch (RestException e) {
+            e.printStackTrace();
+        }
+
+        return list;
+    }
+    
+    public void deleteTemplate(DomainTemplate template) { // 12
+        if (template == null) {
+            System.out.println("Template does not exist");
+        } else {
+            System.out.println("Template " + template.getName() + " will be deleted");
+            try {
+                template.delete(); // 13
+            } catch (RestException e) {
+                e.printStackTrace();
+            }
+        }
+    }
+{% endhighlight %}
+
+In Line 9 of the `fetchAllTemplatesForEnterprise` method, we gather all Domain Template children for a given Enterprise into a Fetcher.  Then in Lines 10 and 11, we c
```

Comment:
Don't use a 135.... IP in documentation, use one from https://tools.ietf.org/html/rfc5737

Suggestion: `198.51.100.108`